test/url-rewriting.test.js

```diff
@@ -56,22 +50,18 @@ test('Should not rewrite if the url is the same', (t, done) => {
     }
   })
 
-  fastify.listen({ port: 0 }, function (err) {
-    t.assert.ifError(err)
-    t.after(() => fastify.close())
-    sget({
-      method: 'GET',
-      url: 'http://localhost:' + fastify.server.address().port + '/this-would-404-without-url-rewrite'
-    }, (err, response, body) => {
-      t.assert.ifError(err)
-      t.assert.strictEqual(response.statusCode, 404)
-      done()
-    })
-  })
+  const fastifyServer = await fastify.listen({ port: 0 })
+
+  t.after(() => fastify.close())
```

Comment:
We dont need t.after, just close the instance at the end. Before we had callback logic. Now we await the request etc..

Reply:
Honestly I would still prefer to keep the after.
`t.after` gets executed even if the test fails, while if I put the close at the bottom of the test, and it fails, it never gets executed.

Reply:
yeah you are right on this.